src/ui/components/DialogErrorWithTabs/DialogErrorWithTabs.scss

```diff
@@ -1,20 +1,33 @@
 .dl-dialog-error {
+    $contencDesktopMaxWidth: 630px;
+    $contentSmallScreenWidth: 300px;
     // TODO: this is an private api and may silently changed. We should use local styles
     --yc-tabs-space-between: 20px;
 
     $class: &;
 
+    width: 100%;
+    max-width: $contencDesktopMaxWidth;
+
     &__content {
-        width: 630px;
-        @media (min-width: 320px) and (max-width: 480px) {
-            width: 300px;
+        width: $contencDesktopMaxWidth;
+        @media (max-width: 480px) {
```

Comment:
Would it effect on mobile view of dialog content?

cc @mournfulCoroner

Reply:
I would, currently if screen size is lesser than 320px, modal will jump back to 630px, i suggested that it would be better to remove 'min-width' breakpoint for this case cause even smaller(rear case for 2024) < 320px screen resolution modal will be bigger than on 320px to 480px